linkify-item-names.user.js

```diff
@@ -132,6 +137,16 @@ function linkify_items_in_syndicate_campaign_rewards() {
     }
 }
 
+function store_item_params(slug) {
+    var item_type = $('.data-list>.type>span');
+    if (item_type.length && (item_type.html().toLowerCase() == 'armor' || item_type.html().toLowerCase() == 'weapon')) {
+        var piercing = $('.data-list>.piercing-damage>span').html();
+        var impact = $('.data-list>.impact-damage>span').html();
+        var energy = $('.data-list>.energy-damage>span').html();
```

Comment:
Hmm. We _could_ refactor these 3 jQuery calls into a single multi-result query + `.each(`_[...]_`)` call, but having simpler code (as it is now) is probably better than shaving off maybe fractions of a millisecond while increasing code complexity.